docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md

```diff
@@ -189,44 +205,43 @@ PING example.com (192.168.100.123) 56(84) bytes of data.
 rtt min/avg/max/mdev = 0.054/0.067/0.080/0.013 ms
 ```
 
-### Verify KubeFATE service
 When `example.com` well set, KubeFATE service version can be shown,
 ```
-kubefate@machine:~/demo$ kubefate version
+kubefate@machine:~/kubefate$ kubefate version
 * kubefate service version=v1.4.1
 * kubefate commandLine version=v1.4.1
 ```
+Note: The `kubefate` CLI can only work in the same directory of config.yaml
```

Comment:
The version number here should be v1.4.3.

Reply:
Thanks for the finding! Fixed.